src/test/groovy/pl/tomaszdziurko/jvm_bloggers/view/login/attack/stream/BruteForceLoginAttackMailSubscriberSpec.groovy

```diff
@@ -0,0 +1,68 @@
+package pl.tomaszdziurko.jvm_bloggers.view.login.attack.stream
+
+import org.apache.commons.lang3.tuple.ImmutablePair
+import org.apache.commons.lang3.tuple.Pair
+import pl.tomaszdziurko.jvm_bloggers.mailing.MailSender
+import rx.Observable
+import spock.lang.Specification
+import java.lang.Void as Should
+
+/**
+ * @author Adam Dec
+ */
+class BruteForceLoginAttackMailSubscriberSpec extends Specification {
+
+    MailSender mailSender;
+    BruteForceLoginAttackMailSubscriber subscriber;
+
+    def setup() {
+        mailSender = Mock(MailSender)
+        subscriber = new BruteForceLoginAttackMailSubscriber(mailSender, "admin@pjug.pl")
+    }
+
+    Should "Should request 1 element from emitter on start"() {
```

Comment:
We agreed to `def "Should..." ` approach ealier. The file still uses wrong test case naming convention.